.gitlab/image_build/docker_linux.yml

```diff
@@ -20,6 +20,8 @@
     - docker build --build-arg CIBUILD=true $BUILD_ARG --file $BUILD_CONTEXT/$ARCH/Dockerfile --pull --tag ${TARGET_TAG}-unsquashed $BUILD_CONTEXT
     - docker-squash ${TARGET_TAG}-unsquashed -t ${TARGET_TAG}
     - docker push $TARGET_TAG
+  # Workaround for temporary network failures
+  retry: 3
```

Comment:
Maximum seems to be 2 https://docs.gitlab.com/ee/ci/yaml/#retry

Reply:
I changed it to 2.